src/scintillator/loader.js

```diff
@@ -12,34 +11,33 @@ const log = debug('scintillator:loader')
 
 // TODO [#637]: Convert the `load` function to async function (instead of using `co`) in src/scintillator/loader.js
 // See issue #575 for more details.
-export function load(xmlPath, progress) {
-  return co(function*() {
-    log('load XML from %s', xmlPath)
-    let $xml = yield loadXml(xmlPath)
+export async function load(xmlPath, progress) {
+  // return co(function* () {
```

Comment:
Since we are already using Git, this line could be removed; no need to add a comment. :wink: